ng2-components/ng2-alfresco-core/src/services/AlfrescoAuthentication.service.ts

```diff
@@ -20,16 +20,14 @@ import { LogService } from './log.service';
 import { SettingsService } from './settings.service';
 import { StorageService } from './storage.service';
 import { AlfrescoApiService } from './alfresco-api.service';
-import { AuthService } from './auth.service';
 
 /** @deprecated AlfrescoAuthenticationService is deprecated. Use AuthService instead */
 @Injectable()
-export class AlfrescoAuthenticationService extends AuthService {
+export class AlfrescoAuthenticationService {
```

Comment:
Yes, having "extends" is wrong, I was thinking over injecting AuthService into AlfrescoAuthenticationService, i.e.

```ts
@Injectable()
export class AlfrescoAuthenticationService {
  constructor(private authService: AuthService) {}

  login(username: string, password: string) {
     return this.authService.login(username, password);
  }
}
```